src/renderer/ui/components/modals/GoToModal.js

```diff
@@ -4,6 +4,33 @@ import { findDOMNode } from 'react-dom';
 import Dialog from './ThemedDialog';
 import FlatButton from 'material-ui/FlatButton';
 import TextField from 'material-ui/TextField';
+import request from 'request';
+
+// Initialize the global Logger to forward to the main process.
+window.Logger = remote.getGlobal('Logger');
```

Comment:
The Logger should already be pulled in to the `renderer` process, I'd like to limit `remote` calls.  Also in general don't inject something into a global that is only being used in this file.